src/RoutingControllersOptions.ts

```diff
@@ -55,7 +55,7 @@ export interface RoutingControllersOptions {
     /**
      * Global class-validator options passed during validate operation.
      */
-    validationOptions?: ValidationOptions;
+    validationOptions?: ValidatorOptions;
```

Comment:
I don't like `ValidationOptions` and `ValidationOptions` in class-validator package. Good to rename then however I didnt find a good name for both when was creating them